src/wasm/kotlin/src/arcs/WasmJsInterop.kt

```diff
@@ -183,6 +194,6 @@ external fun write(msg: WasmString)
 external fun flush()
 
 fun log(msg: String) {
-    write(msg.toWasmString())
-    flush()
-}
+  write(msg.toWasmString())
+  flush()
+}
```

Comment:
Both newlines and whitespace indent seem like something that could/should be in our lint config. If they're not, maybe we should file an issue to fix that.